src/adapters/poi/idunn_poi.js

```diff
@@ -103,6 +103,10 @@ export default class IdunnPoi extends Poi {
     } catch (err) {
       if (err === 404) {
         return;
+      } else if (err === 0) { // When the OPTIONS request is rejected, the error is 0 and not 405
+        console.warn('Headers aren\'t allowed, sending query without them...');
```

Comment:
Just wondering about this: is there a logger I should use or just this warn is fine?

Reply:
`console.warn` is fine with me, especially because this case is supposed to happen only in dev envs.